File: demo-greeting-spring-api/src/main/java/com/demo/demogreetingspringapi/Greeting.java
package com.demo.demogreetingspringapi;

public class Greeting{
    private final long greetingID;   
    private final String greetingContent;

    public Greeting( long greetingID, String greetingContent){
        this.greetingID = greetingID;
        this.greetingContent = greetingContent;
    }

    public long getGreetingID(){ return this.greetingID; }
    public String getGreetingContent(){ return this.greetingContent; }
}

File: demo-greeting-nodejs-api/greeting.js
"use strict";

class Greeting{

    constructor(greetingID, greetingContent){
        this.greetingID = greetingID;
        this.greetingContent = greetingContent;
    }

    getGreetingID(){ return this.greetingID; }
    getGreetingContent(){ return this.greetingContent; }

    static getAndIncrementCounter(){ 
        Greeting.greetingCounter++;
        return Greeting.greetingCounter; 
    }

};

module.exports = Greeting;